scripts/variantstore/docs/RUNNING_EXOMES_ON_GVS.md

```diff
@@ -0,0 +1,60 @@
+# Running Exomes on GVS
+
+This document describes the changes necessary to run exome gVCFs through the GVS workflow. The changes needed to run exomes primarily involve using different parameters.
```

Comment:
This might not be consistent with other docs in this folder, but could you add a sentence at the top clarifying that this is an internal doc and pointing to the beta docs folder for any beta user that ends up in this folder by accident?